### ISSUE TITLE
Buttons in controls bar should be 44px squares

### ISSUE THREAD

**Issue by pixelipo** [nextcloud/server#7481]
Currently, there is a lot of inconsistency in the buttons places on the #controls bar - most of them are 40px wide and 36px, but some apps (e.g. Deck uses 38x36px).

Design guidelines require a minimum (and optimal?) size for anything clickable to be 44px both width and height, so we should update these buttons.

It will also require the entire bar to become larger - currently it's 44px hight, but buttons need some margin so it should become 50px high.

**Comment by jancborchardt** [comment]
(Untagging it from good first / starter issue though cause changing the height of the controls bar etc has wider-reaching implications and needs to be well-tested.)

**Comment by skjnldsv** [comment]
@danxuliu did you fixed it in the latest gallery?

**Comment by danxuliu** [comment]
@skjnldsv 
>  did you fixed it in the latest gallery?

Not yet, but I intended to fix some issues with the buttons in the controls bar of the gallery and this was one of them ;-)

However, based on my initial tests, in my opinion making those buttons 44x44px is visually too much. An alternative would be to keep them at 36x36px, but provide an invisible clickable area around them of 44x44px. But the problem with that approach is that, although in a touchable interface it would work nicely, when using the mouse it would be surprising to click outside the visual borders of the button and still trigger a click on the button.

So... any idea to have nicely sized buttons that work fine with both a touchable screen and a mouse? :-) This should be taken into account too in other areas, for example in the details view of the Files app, which currently does not seem to be very touch friendly.

**Comment by skjnldsv** [comment]
@danxuliu I'm fine with 44px imho. Especially sine screens tend to be bigger and bigger, the pixel size keeps getting smaller with the density increase. I'm not sure 44px is still considered as "big" nowadays :)

**Comment by danxuliu** [comment]
@skjnldsv I do not know what is considered "big" nowadays; what I know is that in a 1920x1200 24" screen they look too bulky for my taste ;-) (although it may be just due to being used to the previous size)

![gallery-controls-buttons-44px](https://user-images.githubusercontent.com/26858233/37109029-2575244c-2239-11e8-8c21-0a4cfbe68a8d.png)

In any case it is just a matter of personal preference, so given that it improves the accessibility and it does not cause any surprising behaviour depending on the pointing device I am fine with changing them to 44px.

**Comment by skjnldsv** [comment]
I need to see before having an opinion. Blank statement won't really help us I think here :)
Maybe you're right and we should reduce it a bit :)

**Comment by MorrisJobke** [comment]
Most likely nothing for 14 -> moving to 15.

**Comment by MorrisJobke** [comment]
Ref #10617

**Comment by MorrisJobke** [comment]
Moving to 15

**Comment by marcoambrosini** [comment]
For the 1st option, I'd go with either tertiary or secondary buttons:

<img width="426" alt="Screenshot 2021-12-02 at 14 31 08" src="https://user-images.githubusercontent.com/26852655/144431756-d152c92e-0a2d-4685-be3f-467ae8fb9153.png">

About the size concerns, the whole files top-bar can grow to 60px, so that there are adequate margins around the elements. By making the topbar 16px taller, even if the buttons increase in size it wouldn't feel so forced.

<img width="578" alt="Screenshot 2021-12-02 at 14 35 51" src="https://user-images.githubusercontent.com/26852655/144432429-4effc3da-d76e-46f4-a8c9-ca0d133dd153.png">

For the share I'd keep the tertiary. The icon itself should be 20px, not 16px though

**Comment by jancborchardt** [comment]
Sounds good @marcoambrosini – cc @skjnldsv probably also something for the Vue rewrite?

**Comment by skjnldsv** [comment]
Done with 28
